### PR TITLE
Release of DBT template to BETA

### PR DESCRIPTION
**Pull request by MonikaFeigler**

### CODE REVIEW

**Review by TomNett — COMMENTED** [review]
"This is a demo data pipeline with dbt. It extracts seed data, run dbt to model data and export it the final destination"

"With dbt, every data analyst, data scientist and data engineer can build their own data pipelines without relying on others."

**Comment by MonikaFeigler** [comment]
@TomNett thank you, where do you want to see it exactly? readme.md?

**Comment by TomNett** [comment]
Short a long description :-) @MonikaFeigler

File: keboola-dbt-tutorial/v1/src/extractor/keboola.ex-http/dbt-seed-files/rows/seed-beers-csv/config.jsonnet [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+  parameters: {
+    path: "seed_beers.csv",
```
Comment (TomNett) [inline keboola-dbt-tutorial/v1/src/extractor/keboola.ex-http/dbt-seed-files/rows/seed-beers-csv/config.jsonnet:3]: extrakce pada pri stahovani tohoto souboru na erroru ze bucket jiz existuje, druhy je ok. @MonikaFeigler 

Component keboola.ex-http job 31801149 ended in error: Component keboola.ex-http job 31801150 ended in error: Cannot upload file "seed_beers" to table "in.c-keboola-ex-http-28681838.seed_beers" in Storage API: The bucket keboola-ex-http-28681838 already exists.

Comment (MonikaFeigler) [reply keboola-dbt-tutorial/v1/src/extractor/keboola.ex-http/dbt-seed-files/rows/seed-beers-csv/config.jsonnet:3]: I'll check it

Comment (MonikaFeigler) [reply keboola-dbt-tutorial/v1/src/extractor/keboola.ex-http/dbt-seed-files/rows/seed-beers-csv/config.jsonnet]: fixed

**Comment by MonikaFeigler** [comment]
@TomNett the extration was fixed, but it started to fail on something what was working before and I do not understand why. 
Job "938442856" ended with user error: "1 of 7 ERROR creating view model WORKSPACE_938442904.beers ..................... [ERROR in 1.05s] 2 of 7 ERROR creating view model WORKSPACE_938442904.breweries ................. [ERROR in 1.05s] Database Error in model beers (models/beers/beers.sql) 002003 (02000): SQL compilation error: Schema 'SAPI_9389."in.c-keboola-ex-http-28681838"' does not exist or not authorized. compiled SQL at target/run/beer_analytics/models/beers/beers.sql Database Error in model breweries (models/beers/breweries.sql) 002003 (02000): SQL compilation error: Schema 'SAPI_9389."in.c-keboola-ex-http-28681838"' does not exist or not authorized. compiled SQL at target/run/beer_analytics/models/beers/breweries.sql"

checking with support

**Comment by TomNett** [comment]
Thanks! @MonikaFeigler regarding the error - is this error on dbt transformation itself? Has this project where you run it activated the feature Read-only role? If not that is for sure the issue as I have seen it before, At the end of January we should roll out the read-only role everywhere,.

**Comment by MonikaFeigler** [comment]
@TomNett you are right. Wrong project for testing. My bad.

**Comment by MonikaFeigler** [comment]
@TomNett everything should be fixed and work as expected.

**Comment by TomNett** [comment]
great! @MonikaFeigler can we please also add google drive data destination for easier first use?

**Comment by MonikaFeigler** [comment]
@TomNett Google Drive destinace není asi uplne ideální na přidání. Zkoušela jsem to a když chceš použít google drive potřebuju od uživatele následující:
1. musí si na svůj Google Drive vytvořit novou složku a nám říct, jaké má ID, což uživatel zjistí z url (neumíme si totiž přes CLI sami zjistit ID My Drive složky)
2. potom si pro každou tabulku, kterou chce exportovat na drive musí předvytvořit file a dát nám do inputů file ID, opět z zjistitelné z url -> kolik výstupních tabulek tolik bude file IDs, které nám musí poskytnout ve správném pořadí 

Rozhodně to není na jednoduché použití. Je to cca podobné jako s google drive, kde si musíš na svůj drive zkopírovat google sheet poskytnutý keboolou.

**Comment by MonikaFeigler** [comment]
@TomNett tak co s tou šablonou? releasneme to nebo ne?

**Comment by TomNett** [comment]
@MonikaFeigler ano, nekomplikujme to a pojdme ji releasnout. Diky

**Comment by MonikaFeigler** [comment]
@TomNett můžeš tedy approvnout toto PR, prosím?

**Review by TomNett — APPROVED** [review]

**Review by jordanrburger — APPROVED** [review]

**Pull request merged**